src/main/java/org/synyx/urlaubsverwaltung/security/SecurityWebConfiguration.java

```diff
@@ -48,9 +50,33 @@ SecurityFilterChain actuatorSecurityFilterChain(final HttpSecurity http) throws
     }
 
     @Bean
-    SecurityFilterChain webSecurityFilterChain(final HttpSecurity http, DelegatingSecurityContextRepository securityContextRepository) throws Exception {
+    @Order(3)
+    SecurityFilterChain calendarSecurityFilterChain(final HttpSecurity http) throws Exception {
+        return http
+            .securityMatchers(configurer ->
+                configurer
+                    .requestMatchers(GET, "/web/company/persons/*/calendar")
+                    .requestMatchers(GET, "/web/departments/*/persons/*/calendar")
+                    .requestMatchers(GET, "/web/persons/*/calendar")
+            )
+            .authorizeHttpRequests(requests ->
+                requests
+                    .requestMatchers(GET, "/web/company/persons/*/calendar").permitAll()
+                    .requestMatchers(GET, "/web/departments/*/persons/*/calendar").permitAll()
+                    .requestMatchers(GET, "/web/persons/*/calendar").permitAll()
+                    .anyRequest().authenticated()
+            )
+            .csrf(AbstractHttpConfigurer::disable)
```

Comment:
Why do we disable csrf here and not before?

Reply:
We can only disable csrf for the whole security filter chain. And now we can because it is separated.